inlong-dashboard/src/metas/sinks/defaults/Elasticsearch.ts

```diff
@@ -92,6 +92,28 @@ export default class ElasticsearchSink
   @I18n('meta.Sinks.ES.IndexName')
   indexName: string;
 
+  @FieldDecorator({
+    type: 'input',
+    rules: [{ required: true }],
+    props: values => ({
+      disabled: [110, 130].includes(values?.status),
+    }),
+  })
+  @ColumnDecorator()
+  @I18n('meta.Sinks.ES.PrimaryKey')
```

Comment:
The `PrimaryKey` for Elasticsearch is so strange, as I know, Elasticsearch does not have a concept of primary key, see here: https://discuss.elastic.co/t/get-data-primary-key/235128

If needed, the user can specify the `_id` field, otherwise, Elasticsearch will auto-generate one for each document.